scripts/jenkins/test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+export GOPATH=$WORKSPACE
+eval "$(gvm $GO_VERSION)"
+go get -v -u github.com/jstemmer/go-junit-report
+go get -v -u code.google.com/p/go.tools/cmd/cover
```

Comment:
```suggestion
```

I think the install instructions for gocover-cobertura are out of date. "go tool cover" is part of the Go toolchain, no need to install anything from go.tools. Just keep the line below to install gocover-cobertura